node/network/bridge/src/lib.rs

```diff
@@ -172,11 +172,11 @@ impl metrics::Metrics for Metrics {
 	) -> std::result::Result<Self, prometheus::PrometheusError> {
 		let metrics = MetricsInner {
 			peer_count: prometheus::register(
-				prometheus::GaugeVec::new(
-					prometheus::Opts::new(
+				prometheus::HistogramVec::new(
+					prometheus::HistogramOpts::new(
 						"polkadot_parachain_peer_count",
-						"The number of peers on a parachain-related peer-set",
-					),
+						"Number of connected peers across all peer sets",
+					).buckets(vec![0.0, 10.0, 20.0, 30.0, 40.0, 50.0, 60.0, 70.0, 80.0, 90.0, 100.0]),
```

Comment:
peer count is 900 on Kusama, not sure those buckets are useful.

Reply:
Yeah, you are right, these make sense only for parachain validators.

It will actually be more precise, since the Gauge samples the instant values while now we have buckets to count more than that.

Reply:
1k for validators and 10-100 for collators. So there's no one-size-fits-all bucket